### ISSUE TITLE
Submodule cmake-common should be included in the release package

### ISSUE THREAD

**Issue by yarda** [HBPVIS/Servus#102]
Submodule cmake-common from the Eyescale should be included in the release tar.gz package, otherwise it's quite hard to package it downstream. I am trying to get this package into Fedora.

**Comment by umlaeute** [comment]
I agree that it would be great if there were usable tarballs that do not require `git` and network-access when building.
